.github/workflows/ci.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+          allow-prereleases: true
```

Comment:
I left this here so that we don't forget, but I could also remove it and it to the draft PR that adds `3.12` back if you prefer:
```suggestion
```

Reply:
I think it's fine to leave it in